realm/realm-library/src/main/java/io/realm/internal/RealmCore.java

```diff
@@ -42,7 +43,12 @@ public static boolean osIsWindows() {
         return (os.contains("win"));
     }
 
-    /**
+    public static boolean osIsDarwin() {
+        String os = System.getProperty("os.name").toLowerCase(Locale.getDefault());
```

Comment:
Isn't it dangerous to use the default locale as that might be anything Shouldn't this be `Locale.US` or something similar where we know how the casing rules work?